scripts/control_device.py

```diff
@@ -273,11 +287,27 @@
     type=int,
     help="Shutter position percentage",
 )
+set_shutter_position_parser.add_argument(
+    "-x",
+    "--index",
+    required=False,
+    type=int,
+    default=0,
+    help="select shutter number id to operate",
+)
```

Comment:
What does the shutter number ID stand for?

Reply:
Basically, that is the circuit number I will update the help string

Reply:
it is done.

Reply:
You've changed "select shutter number ID to operate" to "select circuit number to operate." what's the difference?
Maybe it's just my lack of familiarity with the new Switcher devices; perhaps explain in a comment what "shutter number id" / "circuit number" means to help me understand.

Reply:
They are the same..
That means which shutter number to operate..
Runner S11 has 1 shutter and 2 lights so the possibilities are shutter 0, light 0 and light 1
Runner S12 has 2 shutters and 1 light so the possibilities are shutter 0, shutter 1 and light 0

I think you are the one that circuit number made more sense so I just aligned that here as well.

Reply:
Oh, OK, I see.